templates/desktop/developers.html

```diff
@@ -4,108 +4,136 @@
 
 {% block meta_description %}Learn why Ubuntu is the ideal platform for developers.{% endblock %}
 
-{% block meta_copydoc %}https://docs.google.com/document/d/1VJ3xT2ViC-CvL0UmsAUzpYFmvpuqUP4AbpnlPm8tqOA/edit{% endblock meta_copydoc %}
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1VJ3xT2ViC-CvL0UmsAUzpYFmvpuqUP4AbpnlPm8tqOA/edit
+{% endblock meta_copydoc %}
 
-{% block body_class %}is-paper{% endblock body_class %}
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
 
 {% block content %}
 
-{# 0 -180px #}
+  {# 0 -180px #}
 
-<section class="p-strip is-shallow u-no-padding--bottom">
-  <div class="p-section">
-    <div class="row--50-50">
-      <div class="col">
-        <h1>Ubuntu Desktop <br aria-hidden/>for developers</h1>
-      </div>
-      <div class="col">
-        <p>Whether you're an app or web developer, engineering manager, data scientist, or AI/ML researcher &mdash; Ubuntu is your ideal workstation.</p>
-        <a href="/download/desktop" class="p-button--positive">Get Ubuntu now</a>
+  <section class="p-strip is-shallow u-no-padding--bottom">
+    <div class="p-section">
+      <div class="row--50-50">
+        <div class="col">
+          <h1>
+            Ubuntu Desktop
+            <br aria-hidden="true" />
+            for developers
+          </h1>
+        </div>
+        <div class="col">
+          <p>
+            Whether you're an app or web developer, engineering manager, data scientist, or AI/ML researcher &mdash; Ubuntu is your ideal workstation.
+          </p>
+          <a href="/download/desktop" class="p-button--positive">Get Ubuntu now</a>
+        </div>
       </div>
     </div>
-  </div>
-  <div class="p-section">
-    <div class="u-fixed-width">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/1819c545-woman-working.jpg",
-        alt="",
-        width="2464",
-        height="1052",
+    <div class="p-section u-fixed-width">
+      {{
+        image(
+        url="https://assets.ubuntu.com/v1/bbe34b80-woman-working.jpg",
+        alt="Woman working",
```

Comment:
Let's improve it a bit: "Developer sitting in front of her computer in an office"